mmdet/models/utils/conv_upsample.py

```diff
@@ -0,0 +1,65 @@
+import torch.nn.functional as F
+from mmcv.cnn import ConvModule
+from mmcv.runner import BaseModule, ModuleList
+
+
+class ConvUpsample(BaseModule):
+    """ConvUpsample performs 2x upsampling after Conv.
+
+    This module performs 2x upsampling after Conv. Upsampling will be applied
```

Comment:
The description is inaccurate. Because it performs more than 2x upsamples if num_upsample > 1